static/docs/command-reference/remote/list.md

```diff
@@ -47,7 +47,7 @@ Let's for simplicity add a default local remote:
 While the term may seem contradictory, it doesn't have to be. The "local" part
 refers to the machine where the project is stored, so it can be any directory
 accessible to the same system. The "remote" part refers specifically to the
-project/repository itself.
+project/repository itself. Read "local, but external" storage.
```

Comment:
not sure I get the value of this addendum :)

Reply:
I was just thinking the term "local remote" (storage) should really be called "local external storage" as it is not remote in any way. Then I remembered about these notes explaining the term and added that last sentence... Definitely not something essential, I can remove it it if you think it doesn't help.

Reply:
up to you, was just trying to understand the logic behind it

Reply:
I like it 😬

Reply:
👍 😄